app/models/payment.rb

```diff
@@ -4,23 +4,23 @@ class Payment < ApplicationRecord
 
   self.primary_key = :token
 
-  attr_accessor :issuer, :payment_uri, :message, :payconiq_qrurl, :payconiq_deeplink
+  attr_accessor :issuer, :payment_uri, :message
 
   validates :description, presence: true
   validates :amount, presence: true, numericality: true
   validates :status, presence: true
   validates :payment_type, presence: true
 
   enum status: { failed: 0, in_progress: 1, successful: 2 }
-  enum payment_type: { ideal: 0, payconiq_online: 1, payconiq_display: 2, pin: 3 }
+  enum payment_type: { ideal: 0, pin: 3 }
```

Comment:
Good to see the variants still retaining their original numbers for backwards-compatibility,  but be careful